libkbfs/block_disk_store.go

```diff
@@ -54,6 +55,9 @@ import (
 //           May be missing, but should be present when data is.
 //   - refs: The list of references to the block, encoded as a serialized
 //           blockRefInfo. May be missing.
+//   - f:    Presence of this file indicates the block has been successfully
```

Comment:
can you put this flag in `blockRefInfo` instead? It would be a shame to add one more file per block, especially since we've run into inode limits.

I guess `blockRefInfo` should maybe then be renamed to `blockInfo` or something. But we're stuck with the filename.